### DISCUSSION TITLE
When visit /canvas-ui the js and css file can not find

### DISCUSSION THREAD

**Discussion by caojianfei** [austintoddj/canvas#1085] [Troubleshooting]
It looks like this:
![微信截图_20220630205409](https://user-images.githubusercontent.com/17742555/176682567-ec952fa3-7825-4af0-89fa-87005ca81e4f.png)

I have already run `npm run dev`:
![微信截图_20220630174628](https://user-images.githubusercontent.com/17742555/176682812-f1e5efbd-8881-4c96-b20a-746df486d11a.png)

**Comment by austintoddj** [comment]
Fixed in an earlier version of Canvas.
